packages/node/lib/process-request.js

```diff
@@ -16,10 +16,13 @@ module.exports = (req, options = {}) => {
     req.headers = removeOtherProperties(req.headers, options.whitelist);
   }
 
-  const postData = {};
+  let postData;
```

Comment:
Per the HAR spec, `postData` should only be present when it contains data. If we don't have a request body, there's no need to add it.

Reply:
Have we been enforcing this historically?

Reply:
@dpim We haven't, no. This'll be a new change.